### PR TITLE
adding minimal FAQ

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

File: nex/faq.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Nex uses [firecracker](https://firecracker-microvm.github.io/) to launch small, very fast and very light weight virtual machines. These machines each run an agent that manages the workload in that machine as well as communications with the Nex node.
+
+The root file system is essentially a snapshot of the operating system disk image that will be used when bringing up the virtual machine. It's like a docker layer, but much more efficient.
+
+## Technical Details
+
+### What are namespaces?
+A namespace is a unit of multi-tenancy. All workloads are scoped to a namespace. If you deploy the `echoservice` workload within the `default` namespace and also deploy it within the `sample` namespace, they will not be treated the same by Nex. 
+
+**Note** - there's nothing Nex can do from preventing your code in both of those services from potentially overlapping, clustering, etc.
```
Comment (bruth) [inline nex/faq.md:53]: Would it be more clear to state that namespaces are not a form of isolation?

File: nex/faq.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### What workload types can I run?
+Nex supports two categories of workloads: **services** and **functions**. Services are long-lived while functions are short-lived and triggered on demand by some stimulus.
+
+Services can be any statically compiled 64-bit Linux binary and functions can be either a **WebAssembly** module or a **JavaScript** exported function. We are exploring possible support for running OCI images as services.
+
+### What is a root file system?
+Nex uses [firecracker](https://firecracker-microvm.github.io/) to launch small, very fast and very light weight virtual machines. These machines each run an agent that manages the workload in that machine as well as communications with the Nex node.
+
+The root file system is essentially a snapshot of the operating system disk image that will be used when bringing up the virtual machine. It's like a docker layer, but much more efficient.
```
Comment (bruth) [inline nex/faq.md:46]:
```suggestion
The root file system is essentially a snapshot of the operating system disk image that will be used when bringing up the virtual machine. It's like a Docker layer, but much more efficient.
```

File: nex/faq.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Running Workloads
+
+### What workload types can I run?
+Nex supports two categories of workloads: **services** and **functions**. Services are long-lived while functions are short-lived and triggered on demand by some stimulus.
+
+Services can be any statically compiled 64-bit Linux binary and functions can be either a **WebAssembly** module or a **JavaScript** exported function. We are exploring possible support for running OCI images as services.
+
+### What is a root file system?
+Nex uses [firecracker](https://firecracker-microvm.github.io/) to launch small, very fast and very light weight virtual machines. These machines each run an agent that manages the workload in that machine as well as communications with the Nex node.
```
Comment (bruth) [inline nex/faq.md:44]:
```suggestion
Nex uses [Firecracker](https://firecracker-microvm.github.io/) to launch small, very fast and very light weight virtual machines. These machines each run an agent that manages the workload in that machine as well as communications with the Nex node.
```

File: nex/faq.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+### Technical Details
+
+* [What are namespaces?](faq.md#what-are-namespaces)
+* [Are my workload configurations secure?](faq.md#are-my-workload-configurations-secure)
+* [What is the difference between run and devrun?](faq.md#what-is-the-difference-between-run-and-devrun)
+
+## General
+
+### What is Nex?
+Nex is an open source, lightweight execution engine that can be run in addition to and alongside NATS. Nex nodes form a universal pool into which you can deploy services and functions in JavaScript, WebAssembly, and any language that compiles to native 64-bit Linux binaries.
```
Comment (bruth) [inline nex/faq.md:25]:
```suggestion
Nex is an open source, lightweight execution engine that runs alongside NATS. Nex *nodes* are run to form a universal pool in which you can deploy services and functions in JavaScript, WebAssembly, and any language that compiles to native 64-bit Linux binaries.
```

File: nex/faq.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+In short, `nex run` is meant for production and real deployments while `nex devrun` is meant for developer iteration loops and testing environments. A regular request to run a workload must contain all of the following:
+
+* An issuer **nkey** that counts as the entity certifying the request
+* A publisher **xkey** that is used as the source of encrypted data
+* A URL indicating the location of the workload that is stored in an object store, in the form of `nats://{BUCKET}/{key}`.
+* A set of environment variables to be passed to the workload when it is run
+* A fixed and properly formatted workload _name_
+* A namespace (logical grouping) in which the workload is to be run.
+
+Manually supplying all this information when you're just trying to develop and test on your local machine is cumbersome, so using `devrun` all you need do is supply a path to the workload binary file (`.wasm`, `.js`, _elf_) and the environment variables, and the `nex` CLI will take care of managing the rest for you, including uploading your file to an object store automatically.
```
Comment (bruth) [inline nex/faq.md:70]:
```suggestion
Manually supplying all this information when you're just trying to develop and test on your local machine is cumbersome, so using `devrun` all you need do is supply a path to the workload binary file (`.wasm`, `.js`, ELF binary) and the environment variables, and the `nex` CLI will take care of managing the rest for you, including uploading your file to an object store automatically.
```

**Review by bruth — COMMENTED** [review]
Looks great. A couple minor things and one clarification question.

File: nex/faq.md (hunk 1 above)
Comment (autodidaddict) [reply nex/faq.md:53]: but they are a form of isolation.. we just don't force network isolation. not sure the best way to convey that.

**Review by bruth — APPROVED** [review]

**Pull request merged**